dbms/tests/queries/0_stateless/00670_truncate_temporary_table.sql

```diff
@@ -0,0 +1,16 @@
+drop table if exists test;
```

Comment:
The keyword should be uppercase

Reply:
Test in the test database

Reply:
Well from what I can tell there isn't a consistency or rationale about this.

Reply:
Sure, but I think it would be better : )